kolibri/plugins/user_profile/assets/src/views/ChangeFacility/index.vue

```diff
@@ -1,62 +1,30 @@
 <template>
 
-  <CoreBase
-    :immersivePage="false"
-    :appBarTitle="coreString('profileLabel')"
-    :immersivePagePrimary="true"
-  >
-    <KPageContainer>
+  <NotificationsRoot>
+    <div class="main-wrapper" :style="wrapperStyles">
```

Comment:
`"main-wrapper"` class doesn't seem to be used

Reply:
fixed, thanks!